### DISCUSSION TITLE
Custom Scripts stopped working with version 3.5.5

### DISCUSSION THREAD

**Discussion by jnc27** [netbox-community/netbox#13209] [Q&A]
My script was working fine with version 3.5.4 and has stopped working with 3.5.5.
Are there any changes or adaptations I need to consider?
Thank you

**Comment by abhi1693** [comment]
You will have to share details on the error for someone to help out.

**Reply by jnc27** [reply]
Can be closed.
It was that I used python modules on that script and forgot to add it into local requirements.
